modules/control/proto/leadlag_conf.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto2";
+
+package apollo.control;
+
+message LeadlagConf {
+  optional double innerstate_saturation_level = 1;
+  optional double alpha = 2;
```

Comment:
Add default value to so the default filter gain == 1

Reply:
Suggestions accepted and new commit uploaded